Plugin/Quote/Model/Quote.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Yotpo\SmsBump\Plugin\Quote\Model;
+
+use Magento\Quote\Model\Quote as QuoteModel;
+use Magento\Quote\Api\Data\AddressInterface;
+
+/**
+ * Class Quote
+ * Plugin to trigger checkout sync
+ */
+class Quote extends AbstractCheckoutTrigger
+{
+    /**
+     * @method afterSetBillingAddress
+     * @param  QuoteModel       $quote
+     * @param  QuoteModel       $result
+     * @param  AddressInterface $address
+     * @return QuoteModel
+     */
+    public function afterSetBillingAddress(QuoteModel $quote, $result, AddressInterface $address = null)
+    {
+        $this->checkoutSync($quote);
+        if(!$this->registry->registry('yotpo_smsbump_quote_set_billing_address_plugin')){
+            $this->registry->register('yotpo_smsbump_quote_set_billing_address_plugin', 1);
+        }
+        return $result;
+    }
```

Comment:
if it has the $result and nothing was changed or used within this context, we should change this method to void and use the return from the original callers instead.

Reply:
also `$address` is not used.